src/main/java/org/jabref/gui/Base.css

```diff
@@ -265,8 +265,8 @@
     -jr-ai-message-user-border: -jr-theme;
     -jr-ai-message-ai: -jr-accent;
     -jr-ai-message-ai-border: -jr-theme;
-    -jr-ai-message-error: -jr-light-red;
-    -jr-ai-message-error-border: -jr-red;
+    -jr-ai-message-error: -jr-error;
+    -jr-ai-message-error-border: #982B1C;
```

Comment:
Can you use `derive` here?

Important for `Dark.css` (which you do not need to adapt then)

Reply:
I haven't understood derive. It seems it make text only lighter

Reply:
https://dlsc.com/2014/07/11/javafx-tip-7-use-css-color-constants-derive-colors/
Derive takes same base color and modifies the hue

Reply:
:laughing: Thank you! Why I haven't thought about negative derives...